package.json

```diff
@@ -29,19 +29,21 @@
     "ripple-keypairs": "^1.0.0"
   },
   "devDependencies": {
+    "@fintechstudios/eslint-plugin-chai-as-promised": "^3.0.2",
     "@types/chai": "4.2.11",
     "@types/google-protobuf": "^3.7.2",
     "@types/mocha": "7.0.2",
     "@types/node": "^14.0.4",
-    "@typescript-eslint/eslint-plugin": "^3.1.0",
-    "@typescript-eslint/parser": "^3.1.0",
-    "@xpring-eng/eslint-config-base": "^0.7.0",
+    "@typescript-eslint/eslint-plugin": "^3.4.0",
+    "@typescript-eslint/parser": "^3.4.0",
+    "@xpring-eng/eslint-config-base": "^0.9.0",
     "chai": "4.2.0",
     "copyfiles": "^2.2.0",
-    "eslint": "^7.0.0",
+    "eslint": "^7.3.1",
+    "eslint-plugin-array-func": "^3.1.6",
     "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-import": "^2.20.0",
-    "eslint-plugin-jsdoc": "^27.0.0",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-jsdoc": "^28.5.1",
```

Comment:
Short summary:
`chai-as-promised`: Tricky extension for Chai that makes Chai/Mocha work better with Promises... but things break if you don't `return` the `chai-as-promsied` assertions from your tests, so this adds linting rules for that.

`eslint-plugin-array-func`: Make sure you're using `Array.from()` and various other array functions correctly.

`eslint-plugin-import 2.22.0`: Waaaay better support for TypeScript and types/interfaces.

`eslint-plugin-jsdoc`: Adds a rule requiring documenting errors with the JSDoc `@throws` keyword.